src/App.jsx

```diff
@@ -1,11 +1,35 @@
-import Content from './components/layout/Content';
-import Layout from './components/layout/Layout';
+import { useState } from 'react';
+import Header from './components/header/Header';
+import DetailsView from './pages/DetailsView';
+import HomeView from './pages/HomeView';
 
 const App = () => {
+  const [searchValue, setSearchValue] = useState('react');
+  const [selectedVideo, setSelectedVideo] = useState(null);
+  const [page, setPage] = useState('home');
+
+  const searchHandler = (value) => {
+    if (page !== 'home') {
```

Comment:
**Optional**: 

I recommend you create a constant with these values so that if you ever need to change any value you only need to do it in one place instead of everywhere it is used

so in a `constants.js` (or similar) file

```js
const PAGES {
  home: 'home',
  details: 'details',
  ... etc
}
```

and on these files you can import that value and then `PAGES.home`

Reply:
Thank you, I'll have it in mind.